conf/nebula-graphd.conf.default

```diff
@@ -84,8 +84,9 @@
 # System memory high watermark ratio, cancel the memory checking when the ratio greater than 1.0
 --system_memory_high_watermark_ratio=0.8
 
+########## metrics ##########
```

Comment:
Better to do it in separate PR, don't modify unrelated code.

Reply:
emm, just a format change